api/Document.json

```
@@ -281,6 +281,40 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "mutable": {
+          "__compat": {
+            "description": "Array value is mutable",
```

Nit: Let's reuse the parent spec URL, as there is no more specific fragment.

```suggestion
            "description": "Array value is mutable",
            "spec_url": "https://drafts.csswg.org/cssom/#dom-documentorshadowroot-adoptedstylesheets",
```